jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/importer/ImporterProcessor.java

```diff
@@ -74,9 +75,15 @@ public void processAndBackupFile(Path filePath, Path backupDir, Path failureDir)
 
       // ODE-559
       boolean success = true;
-      try (InputStream inputStream = new FileInputStream(filePath.toFile())) {
+      try {
+         InputStream inputStream = new FileInputStream(filePath.toFile());
```

Comment:
This should probably be in the else part to avoid instantiating a stream unnecessarily and then abandoning it. Making unnecessary work for garbage collector.

Reply:
So the `GZIPInputStream` constructor takes an `InputStream` as an argument, figured this saves us a little bit of instantiation.

Reply:
Right! Sorry, didn't realize that that GZIP takes InputStream as argument. We're good. Thanks,

Reply:
Thanks!